### DISCUSSION TITLE
show that there are different rooms

### DISCUSSION THREAD

**Discussion by navlisilvan** [EventFahrplan/EventFahrplan#467] [Ideas]
Hi,
we tested Eventfahrplan with our last Event and one feedback of the users was, that when you didn´t use the app before it is not evident that you have to swipe right/left to the other rooms. I was wondering if a scrollbar at the top would help for this.

The other point was, that it would be nice to load the pictures of workshops/speaker in the details view but i can see that would be more difficult because there is no linl of it in the Pretalx-XML

**Comment by johnjohndoe** [comment]
Nice to read that you manage to set the app yourself. :+1: I appreciate your feedback. Please come forward if you stumbled into something regarding the customization guide as well. Is the app deployed somewhere public?

Regarding horizontal scrolling I recently received the same feedback. I believe we could try out how a horizontal scrollbar influences the user experience. There has been work done on [scrollbar topic](https://github.com/EventFahrplan/EventFahrplan/pulls?q=is%3Apr+is%3Aclosed+label%3AScrolling) not long ago. So it would be good to continue there. Do you want to work on this?

Speaker information are mostly available and could be integrated in the app. In the past years there has not been a big demand for this, though. Therefore, I did not invest time in working on the topic. If someone wants to come up with a concept, discuss and implement it then that's very welcome.

**Comment by navlisilvan** [comment]
Hi,
we didn´t manage to upload the app in time to google-play, so we send the link around: https://acrolotl.de/fahrplan.apk
We are still asking our guests in a survey how they got along with the app, maybe then there will be more feedback later.

I would like to support you with all this but i am more an event organizer than a programmer :/
For customizing the app i asked a friend for help.
So tell me how i can help and i will :)
